vector/src/main/res/layout/activity_room_directory_picker.xml

```diff
@@ -21,7 +22,7 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:background="?attr/waiting_background_color"
-        android:visibility="gone">
+        tools:visibility="gone">
```

Comment:
Not good. Result will be:
    android:visibility="gone"
    tools:visibility="visible">